Functional Monitoring documentation update (1 of 5) (#50)
mulesoft/docs-functional-monitoring: modules/ROOT/pages/afm-add-schedule.adoc

:imagesdir: ../assets/images

> You can add a schedule to an existing monitor if either of the following two conditions are true:

Reviewer: "either...is" (either one is):
```suggestion
You can add a schedule to an existing monitor if either of the following two conditions is true:
```

> * You plan to run the test associated with the schedule in a public location, and the allocation of schedules for your Anypoint Platform organization has not been exhausted.

Reviewer: Just to stick with plain present tense (as possible or logical)...:
```suggestion
* You plan to run the test associated with the schedule in a public location, and the allocation of schedules for your Anypoint Platform organization is not exhausted.
```

Reviewer: This helps a bit with readers for whom English is not a native language. Verb tense is so different among various languages!

* You plan to run the test associated with the schedule in a private location.

== Procedure

. Select the monitor in which you want to add a schedule.
. Click *Add Schedule* from the *Schedules* table.

> . In the Add Schedule dialog, specify the following:

Reviewer: If Add Schedule is a visible label, then use bold.

+
* The location in which to run the test
* The interval at which to run the test:
+
|===
|Interval |Required Information |Defintion

|Hourly
|Minutes

> |The number of minutes after each hour when the test runs, beginning 60 minutes after the schedule has been created

Reviewer: I have a couple of questions about the meaning and wording of this sentence.
Does "after each hour" refer to "clock hours" 1, 2, 3...or to 60 minutes passing?
Depending on the answer to that, does this mean that the test begins exactly 60 minutes after the schedule is created (what does that mean? What signifies the completion of the creation?) or that 60 minutes after creation, the test period begins but does not initiate until the specified number of minutes after "the hour" (whether that is the clock hour or the 60 minutes)?

Reviewer: Maybe we need a SHORT example?

Reviewer: In any case, ditto simple present tense:
```suggestion
|The number of minutes, after each hour, that triggers the test, beginning 60 minutes after the schedule is created.
```

Author: Ok cool, I reworded for clarity. Hopefully this is better:
"The minute at which the test begins, starting 60 minutes after you create the schedule"

I don't think we need to include an example since the "minutes" value is a list option in the UI.

|Daily
|Time

> |The time (hour and minutes) in a day when the test runs

Reviewer: ```suggestion
|The time each day (hour and minutes) at which the test begins.
```

|Weekly

> |Day of week, Time

Reviewer: ```suggestion
|Day of the week and time of day 
```
...or does the table column Required Information take values that appear onscreen or in syntax? In other words, is this capitalized and separated by a comma to indicate two separate values indicates somewhere rather than "general" narrative required info?

> |The day of the week and time when the test runs

Reviewer: ```suggestion
|The day of the week and the time of that day that initiates the test
```

Reviewer: Maybe we could add a short example to each of these definitions?
The example would need to be formatted as if in syntax:

The day of the week and the time of that day that initiates the test, for example `monday`, `12:30`

Reviewer: ...and I just thought of something: 12- or 24-hour clock?

Author: These values are actually list options in the UI. As far as I can tell, they're always in 12-hour time.

|Monthly
|Day of month, Time

> |The day of the month and time when the test runs

Reviewer: ditto Daily edits ^

|Advanced
|Cron

> |The cron expression describing when the test runs

Reviewer: Alternative:
```suggestion
|The cron expression for the time that initiates the test
```

|===

> * Set Starting at (min)

Reviewer: If any of "Starting at (min)" is a viewable UI label, format in bold.

Reviewer: Also, this does not look parallel to its peer list items: it looks like an imperative (step)

> * Set configuration

Reviewer: ditto

. Click *Add*.

== Result

> The countdown to the first run of the selected test begins.

Reviewer: ```suggestion
The 60-minute countdown to the first run of the selected test begins.
```

Author: The 60-minute countdown only applies if the user selected the Hourly monitor schedule, and this info is contained in the Hourly interval table row.
